### ISSUE TITLE
sparkline for weekly downloads should support RTL

### ISSUE THREAD

**Issue by userquin** [npmx-dev/npmx.dev#428]
The sparkline should be right to left including the dot animation

**Comment by graphieros** [comment]
most probably related to #430

**Comment by graphieros** [comment]
As for the charts themselves, time direction is treated as logical progression, not writing direction, and rtl users are accustomed to left-to-right time charts.

**Comment by tmchow** [comment]
Implemented in the PR above - reverses the sparkline dataset order in RTL locales so the chart reads right-to-left.

**Comment by TheDevMinerTV** [comment]
Closing as per

> [As for the charts themselves, time direction is treated as logical progression, not writing direction, and rtl users are accustomed to left-to-right time charts.](https://github.com/npmx-dev/npmx.dev/issues/428#issuecomment-3973954949)
